agents/plugins/smart

```diff
@@ -108,6 +108,20 @@ if which smartctl > /dev/null 2>&1 ; then
             continue
         fi
 
+        # Exclude removable media
+        if which udevadm > /dev/null 2>&1; then
+            TMP_FILE=$(mktemp)
```

Comment:
Do we need a temporary file here? Can we not directly pipe the ouput of grep into udevadm?

Reply:
We could pipe it directly from udevadm to grep.
I used the temporary file, because I wasn't sure, how it behaves with the `if` statement.